data/css/modules/layout/_paginatedScrolling.scss

```diff
@@ -0,0 +1,14 @@
+scrollbar.vertical slider {
+    min-height: 100px;
+}
+
+.LayoutPaginatedScrolling {
+    background-color: $background-light-color;
+    .text-button {
+      background-color: $primary-medium-color;
+      color: white;
+      padding: 5px;
+      margin: 5px;
+      border-radius: 10px;
+    }
+}
```

Comment:
Do we have some sort of "spec" for what this button should look like? If not, my only suggestion would be to make it look "similar" to other buttons we use, e.g. "See more categories". Just the general look and the go-down-symbolic icon.

philip is also adding a debug button as part of T12422 [here](https://github.com/endlessm/eos-knowledge-lib/pull/1190/commits/c0a4b916d89d2a32619947e3fcc519ce3066fe33#diff-5c14d6ac60c52df7ee198a1bc42cbca2R68).